wd_capabilities.au3

```diff
@@ -74,68 +76,47 @@
 
 #Region - wd_capabilities.au3 UDF - Global's declarations
-Global Enum _
-		$_WD_CAPS__STANDARD__Type, _
-		$_WD_CAPS__STANDARD__FirstIdx, _
-		$_WD_CAPS__STANDARD__CURRENT, _
-		$_WD_CAPS__SPECIFICVENDOR__ObjectName, _
-		$_WD_CAPS__SPECIFICVENDOR__OPTS, _
-		$_WD_CAPS__COUNTER
-
-Global $_WD_CAPS__API[0][$_WD_CAPS__COUNTER]
+Global $_WD_CAPS__OBJECT
+Global $_WD_NOTATION__MATCHTYPE = '' ; $_WD_CAPS__STANDARD__Type
+Global $_WD_NOTATION__SPECIFICVENDOR = '' ; $s_SpecificOptions_KeyName
 
-Global Const $_WD_KEYS__MATCHTYPES = _ ; this should be RegExpPattern
+Global Const $_WD_KEYS__MATCHTYPES = _ ; this should be RegExpPattern of possible "Match Types"
 		'(?i)\A(alwaysMatch|firstMatch)\Z'
 
-Global $_WD_KEYS__STANDARD = _ ; this should be RegExpPattern
+Global $_WD_KEYS__STANDARD_PRIMITIVE = _ ; this should be RegExpPattern of "JSON_PRIMITIVE" - "a boolean/string/number/null element" that should be placed in STANDARD part of Capabilities JSON structure
 		'(?i)\A(acceptInsecureCerts|browserName|browserVersion|platformName|pageLoadStrategy|setWindowRect|strictFileInteractability|unhandledPromptBehavior)\Z'
 
-Global $_WD_KEYS__STANDARD_OBJECT = _ ; this should be RegExpPattern
+Global $_WD_KEYS__STANDARD_OBJECT = _ ; this should be RegExpPattern of "JSON_OBJECT" - "a dictionary element" that should be placed in STANDARD part of Capabilities JSON structure
 		'(?i)\A(proxy|timeouts)\Z'
 
-Global $_WD_KEYS__STANDARD_OBJECT_ARRAY = _ ; this should be RegExpPattern
+Global $_WD_KEYS__STANDARD_OBJECT_ARRAY = _ ; this should be RegExpPattern of "JSON_ARRAY" - "a list of primitive elements" that should be placed in $_WD_KEYS__STANDARD_OBJECT .... as an inner element of "JSON_OBJECT" - "a dictionary element" in STANDARD part of Capabilities JSON structure
 		'(?i)\A(noproxy)\Z'
 
-Global $_WD_KEYS__SPECIFICVENDOR_STRING = _ ; this should be RegExpPattern
-		'(?i)\A(binary|debuggerAddress|minidumpPath)\Z'
+Global $_WD_KEYS__SPECIFICVENDOR_PRIMITIVE = _ ; this should be RegExpPattern of "JSON_PRIMITIVE" - "a boolean/string/number/null element" that should be placed in SPECIFICVENDOR part of Capabilities JSON structure
+		'(?i)\A(binary|debuggerAddress|detach|minidumpPath|w3c)\Z'
 
-Global $_WD_KEYS__SPECIFICVENDOR_BOOLEAN = _ ; this should be RegExpPattern
-		'(?i)\A(w3c|detach)\Z'
-
-Global $_WD_KEYS__SPECIFICVENDOR_ARRAY = _ ; this should be RegExpPattern
+Global $_WD_KEYS__SPECIFICVENDOR_ARRAY = _ ; this should be RegExpPattern of "JSON_ARRAY" - "a list of primitive elements" that should be placed in SPECIFICVENDOR part of Capabilities JSON structure
 		'(?i)\A(args|extensions|excludeSwitches|windowTypes)\Z'
 
-Global $_WD_KEYS__SPECIFICVENDOR_OBJECT = _ ; this should be RegExpPattern
+Global $_WD_KEYS__SPECIFICVENDOR_OBJECT = _ ; this should be RegExpPattern of "JSON_OBJECT" - "a dictionary element" that should be placed in SPECIFICVENDOR part of Capabilities JSON structure
 		'(?i)\A(env|log|prefs|perfLoggingPrefs|mobileEmulation|localState)\Z'
 
```

Comment:
IMO, these comments are too long to be placed to the right of the variable declaration. If you are going to be this detailed, then I suggest changing them like this --

```autoit
; this should be RegExpPattern of "JSON_PRIMITIVE" - "a boolean/string/number/null element" that 
; should be placed in STANDARD part of Capabilities JSON structure
Global $_WD_KEYS__STANDARD_PRIMITIVE = _ 
		'(?i)\A(acceptInsecureCerts|browserName|browserVersion|platformName|pageLoadStrategy|setWindowRect|strictFileInteractability|unhandledPromptBehavior)\Z'
```

Reply:
Ok

Reply:
take a look here:
[$_WD_KEYS__*** comments](https://github.com/Danp2/au3WebDriver/pull/285/commits/bf28b5c37eaf5dc411c56a128e86b33474ed1c45)